main.js

```diff
@@ -0,0 +1,23 @@
+function daysInYear(year) {
+  try {
+    if (!Number.isInteger(year)) {
+      throw new SyntaxError('exeption');
+    }
+    const daysInFebruary = new Date(year, 1, 29).getDate();
+    if (daysInFebruary === 29) {
+      console.log(366);
+    } else {
+      console.log(365);
+    }
+  }
+  catch (err) {
```

Comment:
Based on the requirements we should `it should throw an exception`. This code prevents the exception from being thrown.